src/main/java/org/spongepowered/api/block/BlockType.java

```diff
@@ -121,4 +122,18 @@
      */
     float getEmittedLight();
 
+    /**
+     * Gets if the block is able to be gathered using the silk touch
+     * enchantment.
+     *
+     * @return If the block can be harvested with silk touch
+     */
+    boolean canHarvestWithSilk();
+
+    /**
+     * Gets the equivalent {@link ItemBlock} for this BlockType.
+     *
+     * @return The equivalent {@link ItemBlock}
+     */
+    ItemBlock getHeldItem();
```

Comment:
@SpaceManiac, have you ever used TMI? There are item blocks for all of those, if you can /give yourself it then it has an ItemBlock. That's the thing you're holding.